design-documents/graph-ql/coverage/Cart.graphqls

```diff
@@ -50,7 +50,7 @@ type VirtualCartItem implements CartItemInterface @doc(description: "Virtual Car
 }
 
 type ConfigurableCartItem implements CartItemInterface {
-    child_sku: String! @doc(description: "SKU of the simple product corresponding to a set of selected configurable options.")
```

Comment:
Is `child_sku` in any released version of Magento?

Reply:
`child_sku` is not currently available in the schema. I suggest we return the product itself instead of just the sku. With only the child_sku, it will require an additional query, and likely it won't be possible to fetch the child product anyway

Reply:
> child_sku is not currently available in the schema. I suggest we return the product itself instead of just the sku. With only the child_sku, it will require an additional query, and likely it won't be possible to fetch the child product anyway

Oh I 100% agree - was just trying to figure out if we were introducing a breaking change or not

Reply:
> I suggest we return the product itself instead of just the sku.

We should pretty much always try to do this - anytime we return an ID referencing an entity we're baking in a 2-request flow for clients